<launch>
    <node name="waypoint_control_server" pkg="waypoint_control" type="waypoint_control_server" output="screen">
        <param name="rate"                 type="double" value="50.0"  />
        <param name="max_acceleration"     type="double" value="1.00"  />
        <param name="max_duty"             type="double" value="0.30"  />
        <param name="max_in_place_duty"    type="double" value="0.30"  />
        <param name="min_in_place_duty"    type="double" value="0.05"  />
        <param name="max_driving_duty"     type="double" value="0.30"  />
        <param name="min_driving_duty"     type="double" value="0.10"  />
        <param name="max_manual_duty"      type="double" value="0.30"  />
        <param name="min_manual_duty"      type="double" value="0.05"  />
        <param name="in_place_k"           type="double" value="0.40"  />
        <param name="driving_kx"           type="double" value="5.00"  />
        <param name="driving_ky"           type="double" value="5.00"  />
        <param name="full_autonomy"        type="bool"   value="false" />
        <param name="voltage_compensation" type="bool"   value="false" />
        <param name="min_voltage"          type="double" value="30.0"  />
        <param name="full_voltage"         type="double" value="42.0"  />
        <param name="max_compensated_duty" type="double" value="0.50"  />
        <param name="battery_filter_k"     type="double" value="0.03"  />
    </node>
</launch>